froide/helper/tasks.py

```diff
@@ -52,3 +53,8 @@ def search_instance_delete(model_name: str, pk: Optional[int]) -> None:
     instance.pk = pk
     instance.id = pk
     registry.delete(instance, raise_on_error=False)
+
+
+@celery_app.task(expires=60 * 60)
```

Comment:
Expiry of 1 hour is likely too much. The task would run at least every couple of minutes or so. The idea is that the task cannot be in the queue for longer than it's scheduled frequency so it doesn't fill up the queue or gets executed in parallel.

Reply:
👍